component/src/test/java/io/siddhi/extension/io/grpc/utils/TestServer.java

```diff
@@ -0,0 +1,82 @@
+package io.siddhi.extension.io.grpc.utils;
+
```

Comment:
License is missing. 
Damth has pointed out the license issue, and its bad you have not fixed that!